src/main/resources/assets/computercraft/lua/rom/programs/copy.lua

```diff
@@ -10,7 +10,9 @@ local sDest = shell.resolve( tArgs[2] )
 local tFiles = fs.find( sSource )
 if #tFiles > 0 then
     for n,sFile in ipairs( tFiles ) do
-        if fs.isDir( sDest ) then
+        if fs.exists( sDest ) == true then
+            printError( "File exists" )
```

Comment:
This error message can be confusing if the destination is a folder, perhaps change it to `"Destination exists"`.